.github/workflows/check_pending_sql.yml

```diff
@@ -4,25 +4,8 @@ on:
 
 jobs:
   check-pending-sql:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

Comment:
I'm against this, because we will not realise when `ubuntu-latest` will point to a different version and build will suddenly start failing.

I'm in favour of always using the latest Ubuntu LTS, it's just with this change we will not know when exactly that happens :)

Reply:
for this it shouldnt matter as it is only checking the SQL file and not building the entire solution, no?

Reply:
true